Engine/EditorUI/Panel/InspectorSubpanel/PanelMaterial.cpp

```diff
@@ -113,13 +122,19 @@ void PanelMaterial::Render(std::shared_ptr<Material> material)
 	{
 		App->resources->Save<Material>(material);
 	}
-	if (material->IsCoreResource())
+	if (is_core || extracted)
 	{
 		ImGui::PopItemFlag();
 		ImGui::PopStyleVar();
 	}
 }
 
+bool PanelMaterial::IsMaterialExtracted(const std::shared_ptr<Material> &material)
+{
+	Metafile * meta = App->resources->resource_DB->GetEntry(material->GetUUID());
+	return meta->version == 0;
```

Comment:
Very tricky this one, but easy way to do it, don't judge me pls xD

Reply:
ò.ó